CHANGELOG.md

```diff
@@ -1,30 +1,31 @@
 # Changelog
 
-* [0.34.0](#0340)
```

Comment:
do we want to delete all these ?

Reply:
I wanted to have a TOC, but realized after v0.34.2 it'll be huge PIA to maintain, so I just removed it.